### PR TITLE
tools: align fpgainfo mac for N5010 with N6000

### PR DESCRIPTION
**Pull request by rchriste1**
Example:
Number of MACs                   : 2
MAC address 0                    : 00:E0:ED:5C:72:E0
MAC address 1                    : 00:E0:ED:5C:72:E1

Signed-off-by: Roger Christensen <rc@silicom.dk>

### CODE REVIEW

File: tools/libboard/board_n5010/board_n5010.c [hunk 1]
```diff
@@ -231,23 +175,105 @@ fpga_result print_sec_info(fpga_token token)
 	return print_sec_common_info(token);
 }
 
+uint64_t macaddress_uint64(const struct ether_addr *eth_addr)
```
Comment (tswhison) [inline tools/libboard/board_n5010/board_n5010.c:178]: Are there standard library API's that can be used for this (accounting for endianness)?

Comment (rchriste1) [reply tools/libboard/board_n5010/board_n5010.c:178]: I am not aware of any standard API. I took the code from board_n6000.c , which was written by @anandaravuri. We might consider moving it to board_common.c to share it between board_5010.c and board_n6000.c.

Comment (tswhison) [reply tools/libboard/board_n5010/board_n5010.c:178]: Sounds like a good refactoring.

Comment (rchriste1) [reply tools/libboard/board_n5010/board_n5010.c]: I have pushed the refactoring.

**Review by tswhison — APPROVED** [review]

**Review by anandaravuri — APPROVED** [review]

**Review by michelleyho — APPROVED** [review]

**Pull request merged**